test/lib/RuleManager.test.ts

```diff
@@ -57,20 +58,20 @@ describe('RuleManager', () => {
 
 			it('Filtering by multiple categories returns any rule in either category', async () => {
 				// Set up our filter array.
-				const filters = [new RuleFilter(RULE_FILTER_TYPE.CATEGORY, ['Best Practices', 'Design'])];
+				const filters = [new RuleFilter(FilterType.CATEGORY, ['Best Practices', 'Design'])];
 
 				// Pass the filter array into the manager.
 				const matchingRules = await ruleManager.getRulesMatchingCriteria(filters);
 
 				// Expect the right number of rules to be returned.
-				expect(matchingRules).to.have.lengthOf(22, 'Exactly 22 rules are categorized as "Best Practices" or "Design"');
+				expect(matchingRules).to.have.lengthOf(23, 'Exactly 23 rules are categorized as "Best Practices" or "Design"');
```

Comment:
Where did the extra rule come from?

Reply:
Really not sure, but I manually checked the correctness of the updated input,  which I just created from a newly-generated PmdCatalog.json.  I assumed you created the first one from PMD 6.21 and we are now running 6.22.

Reply:
But we upgraded to 6.22 a _while_ ago, so shouldn't this have been broken for quite some time? Whatever, probably not worth worrying about.